DESCRIPTION

```diff
@@ -31,6 +31,7 @@ Imports:
     zip (>= 1.1.0)
 Suggests:
     DT (>= 0.13),
+    formatR,
```

Comment:
hey @kartikeyakirar can you run verdepcheck flow to determine the minimal needed version of this package?
<img width="825" alt="verdepcheck" src="https://github.com/insightsengineering/teal.reporter/assets/133694481/e31a636b-84f3-4372-8d27-a921ee6747a4">